regression-test/suites/query_p0/sql_functions/array_functions/test_array_functions.groovy

```diff
@@ -112,6 +112,12 @@ suite("test_array_functions") {
     qt_select "SELECT k1, array_enumerate_uniq(k7) from ${tableName} ORDER BY k1"
     qt_select "SELECT k1, array_enumerate_uniq(k8) from ${tableName} ORDER BY k1"
     qt_select "SELECT k1, array_enumerate_uniq(k10) from ${tableName} ORDER BY k1"
+    qt_select_array_shuffle1 "SELECT k1, k2, array_shuffle(k2, 0), shuffle(k2, 0) from ${tableName} ORDER BY k1"
+    qt_select_array_shuffle2 "SELECT k1, k5, array_shuffle(k5, 0), shuffle(k5, 0) from ${tableName} ORDER BY k1"
```

Comment:
the test case of result maybe add other array function to ensure generate stable results

Reply:
done

Reply:
now I think the implement of array_shuffle is all of ok,
and here I means the function array_shuffle(k2)  in this pr is [3, 2, 1]
Is it possible in other PRs the result is [1,2,3], so can't pass the test?